src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -140,6 +142,40 @@ def values(self) -> ArrayLike:
         """
         return self.to_pandas().values
 
+    @property
+    @index_not_implemented()
```

Comment:
would you prefer I do that instead of use the decorator?

Reply:
I did it this way because I can then add all APIs to the Index documentation (it will have build errors if specified in the documentation but no method exists in the code).

Reply:
We can do both. For those APIs that will be added soon, let's keep what you have done. For those APIs not in our plan yet, we can use the `__getattr__` to raise the error so we can track them in our telemetry too.